Products/ZenModel/tests/testXmlEdges.py

```diff
@@ -21,7 +21,7 @@ def testXMLEdges(self):
         """
         Test that checks correct convertion of edges to an XML file
         """
-        import pdb; pdb.set_trace()
+        #import pdb; pdb.set_trace()
```

Comment:
It's not, but runtests won't finish without manually exiting from pdb with this line intact.
I'll take this change out of this PR for clarity.